api/numaresourcesoperator/v1alpha1/numaresourcesoperator_types.go

```diff
@@ -43,6 +43,8 @@ type NUMAResourcesOperatorSpec struct {
 type NodeGroup struct {
 	// MachineConfigPoolSelector defines label selector for the machine config pool
 	MachineConfigPoolSelector *metav1.LabelSelector `json:"machineConfigPoolSelector,omitempty"`
+	// DisablePodsFingerprinting defines if pod fingerprint should be omitted for the machines belonging to this group
+	DisablePodsFingerprinting *bool `json:"disablePodsFingerprinting,omitempty"`
```

Comment:
Is there a specific case where such a requirement is needed?
Having the ability to globally disable the feature isn't enough?

Reply:
The unit of cluster partitioning for NROP is the `NodeGroup`. When we started the NROP, we wanted to support (but to some extent to discourage) having non-homogeneous config per cluster, so the worker nodes can be partitioned somehow (e.g. with different TM policies).
Up until now, we had an opinionated setup which unconditionally set the configuration, but now we want to allow to disable this setting - which should be enabled by default.
Disabling per-cluster is surely easier, but it goes against the original design.